packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -3,19 +3,27 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
+import { runArithmetic } from '@beyondessential/arithmetic';
 import { Aggregator } from '@tupaia/aggregator';
 import {
   getAggregationsByCode,
   extractDataElementCodesFromFormula,
   fetchAnalytics,
 } from './helpers';
-import { Builder, AggregationSpecs, FetchOptions } from '../types';
+import { AnalyticValue, Builder, AggregationSpecs, FetchOptions } from '../types';
+import { Analytic } from '@tupaia/aggregator/dist/analytics/filterAnalytics';
 
-interface ArithmeticConfig {
+export interface ArithmeticConfig {
   readonly formula: string;
   readonly aggregation: AggregationSpecs;
 }
 
+interface AnalyticalEvent {
+  organisationUnit: Analytic['organisationUnit'];
+  period: Analytic['period'];
+  dataValues: Record<Analytic['dataElement'], Analytic['value']>;
+}
```

Comment:
This is very close to the `event` interface, but cannot classify as `Event` because
1. It lacks some fields, eg `program`, `orgUnitName`
2. `eventDate` in events has a different format than periods.

But it made me thinking whether a structure like this, which is closer to analytics, may have various advantages in relation to the dhis-inspired events.

Other names I considered for this:
* `DimensionalValues` - Probably the most accurate, but hard to pronounce + pluralise
* `EventAnalytics`  - Creates confusion with DHIS "event analytics"
* `EventValues` - Hard to pluralise
* `AggregatedEvent` -Well, as mentioned this is not really an `Event`